public/app/features/dashboard-scene/settings/GeneralSettings.tsx

```diff
@@ -1,16 +1,63 @@
-import React from 'react';
+import React, { ChangeEvent } from 'react';
 
 import { PageLayoutType } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import {
+  SceneComponentProps,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneRefreshPicker,
+  SceneTimePicker,
+  sceneGraph,
+} from '@grafana/scenes';
+import { TimeZone } from '@grafana/schema';
+import {
+  Box,
+  CollapsableSection,
+  Field,
+  HorizontalGroup,
+  Input,
+  Label,
+  RadioButtonGroup,
+  TagsInput,
+  TextArea,
+} from '@grafana/ui';
 import { Page } from 'app/core/components/Page/Page';
+import { FolderPicker } from 'app/core/components/Select/FolderPicker';
+import { t, Trans } from 'app/core/internationalization';
+import { TimePickerSettings } from 'app/features/dashboard/components/DashboardSettings/TimePickerSettings';
+import { DeleteDashboardButton } from 'app/features/dashboard/components/DeleteDashboard/DeleteDashboardButton';
 
+import { DashboardControls } from '../scene/DashboardControls';
+import { DashboardScene } from '../scene/DashboardScene';
 import { NavToolbarActions } from '../scene/NavToolbarActions';
 import { getDashboardSceneFor } from '../utils/utils';
 
 import { DashboardEditView, useDashboardEditPageNav } from './utils';
 
 export interface GeneralSettingsEditViewState extends SceneObjectState {}
 
+const EDITABLE_OPTIONS = [
+  { label: 'Editable', value: true },
+  { label: 'Read-only', value: false },
+];
+
+const GRAPH_TOOLTIP_OPTIONS = [
+  { value: 0, label: 'Default' },
+  { value: 1, label: 'Shared crosshair' },
+  { value: 2, label: 'Shared Tooltip' },
+];
+
+function getRefreshPickerControl(scene: DashboardScene): SceneRefreshPicker | undefined {
```

Comment:
Please add this function to https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/utils/dashboardSceneGraph.ts